ansible/tasks/installOpenwhiskCatalog.yml

```diff
@@ -5,6 +5,7 @@
     catalog_location={{ item.value.location }}
     catalog_repo_url={{ item.value.url }}
     api_host={{ groups['edge'] | first }}
```

Comment:
I know.  that is why api_host should be edge_host and since it is referring to edge {{ groups['edge'] | first }}